security/external_secrets_operator/external-secrets-operator-release-notes.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="external-secrets-operator-release-notes"]
+= {external-secrets-operator} release notes
+include::_attributes/common-attributes.adoc[]
+:context: external-secrets-operator-release-notes
+
+toc::[]
+
+The {external-secrets-operator} is a cluster-wide service that provides lifecycle management for secrets fetched from external secret management systems.
+
+These release notes track the development of {external-secrets-operator-short}.
+
+For more information, see xref:../../security/external_secrets_operator/index.adoc#external-secrets-operator-about[About the {external-secrets-operator}].
+
+[id="external-secrets-operator-release-notes-0-1-0_{context}"]
+== {external-secrets-operator} 0.1.0
+//content to be added
```

Comment:
There isn't anything in here yet. Should this release notes file not be added yet? You could just comment it out in the TOC for now.